api/routes/api_news.py

```diff
@@ -1,18 +1,31 @@
-from flask import Blueprint, jsonify
-from service.api_logic.news_logic import get_news_by_count
+from flask import Blueprint, jsonify, request
+from service.api_logic.news_logic import get_news_by_count, get_latest_sport_news
 from database.session import SessionLocal
+from api.routes.cache import cache
 
 session = SessionLocal()
 news_app = Blueprint('news', __name__)
 
+def make_cache_key():
```

Comment:
can we only have one of those decorators in our code? If it serves the exact same purpose as the one in api_games file we should reuse it by moving it to some other place first and then importing wherever needed.